symengine/add.h

```diff
@@ -69,11 +69,11 @@ class Add : public Basic
 
     virtual vec_basic get_args() const;
 
-    RCP<const Number> get_coef() const
+    inline const RCP<const Number> &get_coef() const
     {
         return coef_;
     }
```

Comment:
@isuruf, per our gitter discussion, indeed they are not different. We should return a reference.